src/lib/Ouroboros/Byron/Proxy/Main.hs

```diff
@@ -121,16 +137,61 @@ import qualified Ouroboros.Network.TxSubmission.Outbound   as Tx.Out
 import           Ouroboros.Storage.ChainDB.API             (ChainDB)
 import qualified Ouroboros.Storage.ChainDB.API             as ChainDB
 
--- | Definitions required in order to run the Byron proxy.
+-- | Configuration used to get a 'ByronProxy'.
+-- 'configFromGenesis' will make one from a genesis configuration.
 data ByronProxyConfig = ByronProxyConfig
-  { -- TODO see if we can't derive the block version data from the database.
-    bpcAdoptedBVData   :: !BlockVersionData
+  { -- | The Byron Logic layer needs the BlockVersionData. Technically this
+    -- is not constant over a whole blockchain: it can be changed by an update.
+    -- However, in cardano-byron-proxy it is only used to determine message
+    -- size limits on Byron protocols, so as long as header, block, and tx
+    -- size limits do not change, it's ok.
```

Comment:
It might be good to add this to top level description of `byron-proxy`, e.g. `README.md` or  a `wiki` page, if devops should be awere of this.